manifests/t/Tableau/Reader/23.3.345/Tableau.Reader.installer.yaml

```
@@ -3,28 +3,35 @@
 
 PackageIdentifier: Tableau.Reader
 PackageVersion: 23.3.345
-Platform:
-- Windows.Desktop
-MinimumOSVersion: 10.0.0.0
-InstallerType: burn
-Scope: machine
-InstallModes:
-- interactive
-- silent
-- silentWithProgress
-InstallerSwitches:
-  Custom: ACCEPTEULA=1
-UpgradeBehavior: install
-ProductCode: '{137f7d83-6439-4c82-9dff-a4d926ad7afb}'
-AppsAndFeaturesEntries:
-- DisplayName: Tableau Reader 2023.3 (20233.23.1017.0948)
-  Publisher: Tableau Software, LLC
-  DisplayVersion: 23.3.345
-  ProductCode: '{137f7d83-6439-4c82-9dff-a4d926ad7afb}'
-  InstallerType: burn
 Installers:
-- Architecture: x64
+- InstallerLocale: en-US
+  Platform:
+  - Windows.Desktop
+  MinimumOSVersion: 10.0.0.0
+  Architecture: x64
+  InstallerType: burn
+  Scope: machine
   InstallerUrl: https://downloads.tableau.com/tssoftware/TableauReader-64bit-2023-3-0.exe
-  InstallerSha256: 660DD062B5957D9D4F23935F80E4CF61EFDC123D1C4996177DD1DD48BC067E6F
+  InstallerSha256: 660dd062b5957d9d4f23935f80e4cf61efdc123d1c4996177dd1dd48bc067e6f

```

```suggestion
  InstallerSha256: 660DD062B5957D9D4F23935F80E4CF61EFDC123D1C4996177DD1DD48BC067E6F
```
I believe the preference is for all-caps hash, but I can't find documentation stating this.

---

@stephengillie Interesting. Winget produces lower hash. Happy to change this in next PRs.

---

Oh, interesting:
```PowerShell
PS C:\ManVal> winget hash .\Installers\124965.htm
InstallerSha256: 8b4856bf204b1cf2991082a2346ec1ca6f056493951b1334c6bd8963d58ff418
PS C:\ManVal> Get-FileHash .\Installers\124965.htm

Algorithm       Hash                                                                   Path
---------       ----                                                                   ----
SHA256          8B4856BF204B1CF2991082A2346EC1CA6F056493951B1334C6BD8963D58FF418       C:\ManVal\Installers\124965.htm
```